pkg/splunk/enterprise/indexercluster.go

```diff
@@ -939,6 +957,16 @@ func (mgr *indexerClusterPodManager) verifyRFPeers(ctx context.Context, c splcom
 	return nil
 }
 
+var GetClusterManagerInfoCall = func(ctx context.Context, mgr *indexerClusterPodManager) (*splclient.ClusterManagerInfo, error) {
```

Comment:
Should we rename this as `GetClusterManagerInfo` vs `GetClusterManagerInfoCall`? Even though its used in mock for UTs?

Reply:
this is standard nomenclature to add `Call` to function pointers

Reply:
We have a few of them in our code, maybe we should change them all at some point.